src/python/dxpy/scripts/dx.py

```diff
@@ -2869,6 +2870,11 @@ def run_one(args, executable, dest_proj, dest_path, input_json, run_kwargs):
                 else:
                     ssh_args = parser.parse_args(['ssh', dxexecution.get_id()])
                 ssh(ssh_args, ssh_config_verified=True)
+    except PermissionDenied as e:
+        if run_kwargs.get("detach") and os.environ.get("DX_RUN_DETACH") == "1" and "detachedJob" in e.msg:
+            print("Unable to start detached job in given project. "
+                  "To disable running detached job by default unset the environment variable DX_RUN_DETACH ('unset DX_RUN_DETACH')")
+        err_exit()
```

Comment:
I'd `raise(e)` to show the actual error that mentions detachedJob license.

Reply:
The actual error is shown as well, as err_exit() shows the error. 

Do you want me to create a new, different exception than the one that really occured?

Reply:
In that case, no need to `raise(e)`.  Thanks Marek!

Reply:
I changed it to raise(e)

but note that this change will not affect anything as err_exit() part of the code calls this:
`    exc = exception if exception is not None else sys.exc_info()[1]
    if isinstance(exc, SystemExit):
        raise exc`
        
+ does some other code that should not affect anything in this case.